modules/go-add-infra-nodes.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="add-infra-nodes_{context}"]
+= Adding Infra Nodes
+
+.Procedure
+
+. Label the existing nodes as Infrastructure usinf the following command:app-name:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra
+----
+. Add `runOnInfra` toggle in the `GitOpsService` Custom Resource (CR) to add infra node selector:
++
+[source,terminal]
+----
+$ oc edit gitopsservice -n openshift-gitops
+----
+. Add `runOnInfra` to spec and set to `true` in the YAML:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Optional: You can also apply taints and isolate the workloads on infrastructure nodes and prevent other workloads from scheduling on these nodes. 
++
+[source,terminal]
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra 
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
+If you have applied taints to the nodes, you can add tolerations in the `GitOpsService` CR:app-name:
++
+[source,yaml]
+----
+spec:
+  runOnInfra: true
+  tolerations:
+  - effect: NoSchedule
+    key: infra
+    value: reserved
+  - effect: NoExecute
+    key: infra
+    value: reserved 
+----
+
+To verify that the workloads are scheduled on infrastructure nodes in the {gitops-title} namespace, click on any of the pods and see the details. The *nodeSelector* and the *tolerations* will be added.
```

Comment:
```suggestion
To verify that the workloads are scheduled on infrastructure nodes in the {gitops-title} namespace, click any of the pod names and ensure that *nodeSelector* and  *tolerations* have been added.
```
- Per IBM SG: do not use `click on`; use "click" instead
- Change `pod names` instead of `pods`, depending on what the user actually has to click in the UI.